src/main/java/de/tum/in/www1/artemis/service/scheduled/cache/statistics/ExamLiveStatisticsDistributedCache.java

```diff
@@ -20,15 +20,15 @@
 import com.hazelcast.nio.serialization.StreamSerializer;
 
 import de.tum.in.www1.artemis.config.Constants;
-import de.tum.in.www1.artemis.domain.exam.monitoring.ExamActivity;
+import de.tum.in.www1.artemis.domain.exam.statistics.ExamActivity;
 
 /**
- * This class represents the cache for a single exam monitoring.
+ * This class represents the cache for a single exam live statistics.
  * <p>
  */
-public class ExamMonitoringDistributedCache extends ExamMonitoringCache implements HazelcastInstanceAware {
+public class ExamLiveStatisticsDistributedCache extends ExamLiveStatisticsCache implements HazelcastInstanceAware {
 
-    private final Logger logger = LoggerFactory.getLogger(ExamMonitoringDistributedCache.class);
+    private final Logger logger = LoggerFactory.getLogger(ExamLiveStatisticsDistributedCache.class);
 
     private static final String HAZELCAST_CACHE_ACTIVITIES = "-activities";
```

Comment:
Should maybe be moved to Constants?

Reply:
There is always the discussion if we should keep the constants on a general place or where we actually use them. So i would leave it as it is.